sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/CatalogSuite.scala

```diff
@@ -858,16 +858,16 @@ class CatalogSuite extends SparkFunSuite {
 
     catalog.alterNamespace(testNs, NamespaceChange.setProperty("property2", "value2"))
     assert(catalog.loadNamespaceMetadata(testNs).asScala === Map(
-      "property" -> "value", "property2" -> "value2"))
+      "property2" -> "value2"))
```

Comment:
This is where the behavior can change based on the catalog implementation, and I don't think we are documenting what's the expected behavior of handling existing properties for `alterNamespace`.